### ISSUE TITLE
The matrix2euler is wrong

### ISSUE THREAD

**Issue by Talegqz** [alisterburt/eulerangles#2]
```python
from eulerangles import euler2matrix, EulerAngleConvention
from eulerangles import matrix2euler
eulers = [[10, 20, 60]]

rotation_matrices = euler2matrix(eulers, axes='ZYX', intrinsic=True, positive_ccw=True)

print(rotation_matrices)
eulers = matrix2euler(rotation_matrices, target_axes='ZYX', target_positive_ccw=True, target_intrinsic=True)
print(eulers)

rotation_matrices2 = euler2matrix(eulers, axes='ZYX', intrinsic=True, positive_ccw=True)
print(rotation_matrices2-rotation_matrices)
```
It can not get right result, could you tell me why?

**Comment by alisterburt** [comment]
Hey,
I'm really sorry about this, this was one of my first projects and I didn't really understand how to test everything properly. I just checked everything and it seems like 5/12 of the extrinsic conversions are actually broken, which is really bad.

I have isolated the bad conversions and made them raise a NotImplementedError rather than returning wrong values (#3) , I will try to figure out exactly what is wrong with these cases and fix them up soon

**Comment by alisterburt** [comment]
#4 is a work in progress PR fixing broken conversions

**Comment by Talegqz** [comment]
ok, I see, thanks for you apply.

**Comment by alisterburt** [comment]
@Talegqz all fixed, updated version 0.1.4 now available on PyPI :rocket:
